integration-tests/ws-trust/src/test/java/io/quarkiverse/cxf/it/ws/trust/CxfWsTrustTest.java

```diff
@@ -83,15 +83,15 @@ public void programmaticSts() throws Exception {
             final QName serviceName = new QName("https://quarkiverse.github.io/quarkiverse-docs/quarkus-cxf/test/ws-trust",
                     "TrustHelloService");
             final URL wsdlURL = new URL(io.quarkiverse.cxf.test.QuarkusCxfClientTestUtil.getServerUrl()
-                    + "/jaxws-samples-wsse-policy-trust/TrustHelloService?wsdl");
+                    + "/jaxws-samples-wsse-policy-trust?wsdl");
```

Comment:
@ffang, @shumonsharif, @famod, any opinion, whether it is fine to generally stop supporting `/my-path/MyService?wsdl`? Note that `/my-path?wsdl` worked before this change. I actually suspect that `/my-path/AnyString?wsdl` worked before too, which sounds like a bug to me that we are hereby fixing. But I admit I have little knowledge about what is normal in the world of CXF and therefore I am asking.